go.mod

```diff
@@ -2,9 +2,20 @@ module github.com/rudderlabs/rudder-server
 
 go 1.18
 
+replace (
```

Comment:
Would it be useful to add a comment here to remind us that at some point, once we upgrade the relevant libraries, these replacements can become outdated and should be removed?

Reply:
Yeah, eventually, we should do that, and we also need to keep the replacement versions up to date.

I am not fun of `replace` solution, and I tried to use it as little as possible. I am considering ways we can avoid it or make it easier to maintain.